src/js/translations/en-US.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Let's add it all in this PR. I think it belongs together.
If you want to do it in a different PR then I think we should have it in a feature branch. Once we all agree on it we can merge everything into master.

Feel free to point to the React-intl repo as it's not going to be obvious for people.